backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/MultiplexTestResult.java

```diff
@@ -0,0 +1,12 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class MultiplexTestResult {
```

Comment:
I wish this wasn't so rigid,  a map of test results would be more extensible for adding more diseases, that way we dont have to keep editing the interface when we add more diseases

Reply:
@nickclyde given that this is more extensible now, could we rename to something like `DiseaseResult`?